opm/parser/eclipse/EclipseState/Schedule/Schedule.hpp

```diff
@@ -107,6 +109,8 @@ namespace Opm
         const Events& getEvents() const;
         const Deck& getModifierDeck(size_t timeStep) const;
         bool hasOilVaporizationProperties() const;
+        const VFPProdTable& getVFPProdTable(int table_id, size_t timeStep) const;
+        const VFPInjTable& getVFPInjTable(int table_id, size_t timeStep) const;
```

Comment:
In the old way, getVFPProdTable() will return `const std::map<int, VFPProdTable>&`, basically it contains all the  VFPPROD tables.  Can we have a function here?
```
const std::map<int, VFPProdTable>& getVFPProdTable(size_t timeStep) const;
```
Basically, I need an easy way to know all the available `table_id`s. With the above function, the downstream change can also be minimal. How do you think?

Similar things can be done in the simulator side, while I think it can be easier to do on the parser side. 

Same with VFPINJ tables.

Reply:
Sure - I will add that method.